src/game/start.py

```diff
@@ -11,14 +12,42 @@ def start_game():
     pieces_list = Piece.create_pieces_list()
 
     game_state = GameState(parameter)
-    if len(game_state.message) > 0:
-        error_message = game_state.message
-
     if len(error_message) > 0:
-        print(error_message)
-        print("""Valid sample : --state='{"grid" : {"A2": 10,"C1":3,"D1":12},"turn" :{"player" : 1,"selected" : 7}}'""")
-
-    display_game(game_state)
+        game_state.message = error_message
+
+    if (len(game_state.message) > 0):
+        game_state.message += """\nValid sample : --state='{"grid" : {"A2": 10,"C1":3,"D1":12},"turn" :{"player" : 1,"selected" : 7}}'"""
+
+    while len(game_state.remaining_pieces) > 0:
+        display_game(game_state)
+        prompt_piece_selection(game_state)
+        game_state.swich_player()
+        display_game(game_state)
+        prompt_piece_location(game_state)
+
+
+def prompt_piece_selection(game_state):
+    value_ok = False
+    while not value_ok:
+        piece = input("Choose the next piece of the opponent : ")
+        try:
+            piece = int(piece)
+            valid_piece = game_state.check_piece_validity(piece)
+            if not valid_piece:
+                game_state.message = "You must choose a number available in the list"
+                display_game(game_state)
+            else:
+                value_ok = True
```

Comment:
Instead of using a boolean to interrupt the loop use `while True` and `break` at the right time